test/install-aws.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+
+BUILD_OS=$(uname 2>/dev/null || echo Unknown)
+
+CMD="aws"
+if [[ -z "$(command -v ${CMD})" ]]; then
+echo "Attempting install of ${CMD}..."
+case "${BUILD_OS}" in
+  Linux)
+    curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+    unzip awscliv2.zip
+    sudo ./aws/install -i
+    rm -rf ./aws
+    rm awscliv2.zip
+    ;;
+  Darwin)
+    curl "https://awscli.amazonaws.com/AWSCLIV2.pkg" -o "AWSCLIV2.pkg"
+    sudo installer -pkg AWSCLIV2.pkg -target /
+    rm -rf ./aws
+    rm awscliv2.zip
```

Comment:
```suggestion
    rm AWSCLIV2.pkg
```

Reply:
Thanks @navidshaikh for looking into this PR.
Thanks for noticing this i will correct it in next commit.